src/github/dto/create-github.dto.ts

```diff
@@ -1,14 +1,9 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsNotEmpty, IsString } from 'class-validator';
+import { IsNotEmpty, IsOptional, IsString } from 'class-validator';
```

Comment:
I don't know if this is necessary but I think the linter is right, and we should remove this 😅 

```suggestion
import { IsNotEmpty, IsString } from 'class-validator';
```